test-data/unit/check-functions.test

```diff
@@ -2085,7 +2085,7 @@ class A(Generic[t]):
         return None
 [builtins fixtures/bool.pyi]
 [out]
-main:5: error: Cannot use a covariant type variable as a parameter
+main:5: error: This usage of this covariant type variable is unsafe as an input parameter.$NLIf this is intentional and you know what you are doing, you can ignore this line with 'unsafe-variance'
```

Comment:
> This usage of this

sounds a bit odd. maybe reword to be like 
> A covariant type variable is unsafe as an input parameter.

Reply:
🙄🙄🙄 out of scope?